web-app/django/VIM/settings.py

```diff
@@ -23,7 +23,8 @@
 IS_PRODUCTION = not IS_DEVELOPMENT
 
 # SECURITY WARNING: don't run with debug turned on in production!
-DEBUG = IS_DEVELOPMENT
+# DEBUG = IS_DEVELOPMENT
+DEBUG = False
```

Comment:
So DEBUG = False is needed to view the 404 page that is served, otherwise django serves its own version. I can comment it and leave the DEBUG=IS_DEVELOPMENT as this is not production version.

Will also update the black-formatting.

Reply:
So the first image is when DEBUG = False. Django is serving what would typically be served on our production server and this is what we want to display because it excludes information related to the backend.
![image](https://github.com/DDMAL/VIM/assets/59708333/35155335-aac4-4de5-9518-a53eefb284f2)

The second image is when DEBUG = True = IS_DEVELOPMENT. When debugging is true, Django will display a standard 404 page with more details regarding the problem it encountered when trying to display the webpage. 
![image](https://github.com/DDMAL/VIM/assets/59708333/df2451bf-315b-4d37-9ff9-01b1e2721f6a)

Reply:
Ok, gotcha